consult.el

```diff
@@ -927,22 +928,26 @@ CAND is the currently selected candidate."
       (let ((start 0)
```

Comment:
The start-of-line check should be performed right in the beginning, just return pos directly.

Reply:
Sure, I guess it could be done that way. I'll change it.